### PR TITLE
chore(docs): document Flink connector

### PR DESCRIPTION
**Pull request by jerrinot**

### CODE REVIEW

File: docs/third-party-tools/flink.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+- A local JDK installation
+- Internet access
+- Docker for running QuestDB
+
+### Connector installation
+- Start QuestDB container image: `docker run -p 9000:9000 -p 9009:9009 questdb/questdb:7.0.1`
+- Download Apache Flink distribution from https://flink.apache.org/downloads/ and unpack it.
+- Download QuestDB Flink connector from [Maven Central](https://repo1.maven.org/maven2/org/questdb/flink-questdb-connector/0.2/flink-questdb-connector-0.2.jar) and place it in the `lib` directory of your Flink installation. 
+- Go to the `bin` directory of your Flink installation and run `./sql-client.sh embedded` to start the Flink SQL console and execute the following commands:
+    ```sql
```
Comment (amyshwang) [inline docs/third-party-tools/flink.md:44]: After Create table there is nothing in questdb web console

File: docs/third-party-tools/flink.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+|------------------|---------|-----------------------|-----------------------------|---------------------------------------------------------------------------|
+| host             | STRING  | localhost:9009        | N/A                         | Host and port where QuestDB server is running                             |
+| username         | STRING  | testUser1             | admin                       | Username for authentication. The default is used when also `token` is set |
+| token            | STRING  | GwBXoGG5c6NoUTLXnzMxw | admin                       | Token for authentication                                                  |
+| table            | STRING  | my_table              | Same as Flink table name    | Target table in QuestDB                                                   |
+| tls              | BOOLEAN | true                  | false                       | Whether to use TLS/SSL for connecting to QuestDB server                   | 
+| buffer.size.kb   | INTEGER | 32                    | 64                          | Size of the QuestDB client send buffer                                    |
```
Comment (amyshwang) [inline docs/third-party-tools/flink.md]: Is send a verb here? I am not sure what this means so please ignore if it does not make sense:
```suggestion
| buffer.size.kb   | INTEGER | 32                    | 64                          | Size of the QuestDB client-sent buffer                                    |
```

**Review by amyshwang — COMMENTED** [review]
As usual something does not work for me...

File: docs/third-party-tools/flink.md (hunk 1 above)
Comment (jerrinot) [reply docs/third-party-tools/flink.md:44]: That's right. This SQL is not executed in QuestDB. It creates a virtual table in Flink. 
The table is linked to QuestDB via the connector. So only when you insert into the virtual table, then Flink will push the data to QuestDB. I hope it makes some sense. but I take this is an indication this part should be clearer.

File: docs/third-party-tools/flink.md (hunk 2 above)
Comment (jerrinot) [reply docs/third-party-tools/flink.md]: "send buffer" is a technical term, it should stay. 

this reminds me to send a PR to improve the Configuration section of the Java client: https://questdb.io/docs/reference/clients/java_ilp/#configuration

File: docs/third-party-tools/flink.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+     buyer        STRING
+    ) WITH (
+    'connector'='questdb',
+    'host'='localhost'
+    );
+  ```
+  Expected output: `[INFO] Execute statement succeed.`
+  
+  This command created a Flink table backed by QuestDB. The table is called `Orders` and has three columns: `order_number`, `price`, and `buyer`. The `connector` option specifies the QuestDB Flink connector. The `host` option specifies the host and port where QuestDB is running. The default port is `9009`. 
+- While still in the Flink SQL console execute: `insert into Orders values (0, 42, 'IBM');`
```
[the remark this answers is not in the document]
Comment (jerrinot) [reply docs/third-party-tools/flink.md]: This looks like a Java version issue. 
can you paste the output of `java --version` and `echo $JAVA_HOME` ?

File: docs/third-party-tools/flink.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+This section shows how to use the QuestDB Flink connector to ingest data from Flink into QuestDB. It uses the SQL interface to interact with Flink. First it creates a table in Flink backed by QuestDB. Then it inserts data into the table. Finally, it queries the data in QuestDB.  
+
+### Prerequisites
+- A local JDK installation
+- Internet access
+- Docker for running QuestDB
+
+### Connector installation
+- Start QuestDB container image: `docker run -p 9000:9000 -p 9009:9009 questdb/questdb:7.0.1`
+- Download Apache Flink distribution from https://flink.apache.org/downloads/ and unpack it.
```
[the remark this answers is not in the document]
Comment (jerrinot) [reply docs/third-party-tools/flink.md]: Kinda. Apache Flink is a project name. And there are different distributions (=packaging) of the project. 

This tutorial uses the distribution created directly by the Apache Foundation. The principles from this tutorial should work in other Flink distributions, but they might differ in directory paths, default configurations, etc. 

It's sort of similar to Kafka: The project name is Apache Kafka and there are various distributions of Kafka: One prepared by the Apache Foundation, there is the Confluent distribution, RedHat has its own Kafka distribution/packaging, etc.

File: docs/third-party-tools/flink.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+title: QuestDB Flink connector
+sidebar_label: Flink
+description: QuestDB ships a QuestDB Flink connector for ingesting messages from Apache Flink via the ILP protocol.
+---
+
+QuestDB ships a
+[QuestDB Flink Sink connector](https://github.com/questdb/flink-questdb-connector)
+for fast ingestion from [Apache Flink](https://flink.apache.org/) into QuestDB. The connector implements the [Table API and SQL](https://nightlies.apache.org/flink/flink-docs-release-1.16/docs/dev/table/overview/) for Flink. 
+
+![Apache Flink logo](/img/guides/flink/flink.svg)
```
[the remark this answers is not in the document]
Comment (jerrinot) [reply docs/third-party-tools/flink.md:13]: is it? it renders for me:
image


it's SVG, but I believe that's widely supported in all browsers. @argshook @insmac can you please confirm it's OK to use SVGs?

[the remark this answers is not in the document]
Comment (argshook) [reply docs/third-party-tools/flink.md:13]: Yep, it's okay

[the remark this answers is not in the document]
Comment (jerrinot) [reply docs/third-party-tools/flink.md:13]: thanks!

File: docs/third-party-tools/flink.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (amyshwang) [reply docs/third-party-tools/flink.md]:
```
➜  ~ java --version
➜  ~ java --version
java 19.0.2 2023-01-17
Java(TM) SE Runtime Environment (build 19.0.2+7-44)
Java HotSpot(TM) 64-Bit Server VM (build 19.0.2+7-44, mixed mode, sharing)
➜  ~ echo $JAVA_HOME

➜  ~ 
```

Something seems wrong? I am using M1 btw

**Review by amyshwang — APPROVED** [review]

**Pull request merged**